src/components/UI/BreadCrumbs.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="breadcrumbs">
+    <button
+      class="breadcrumbs__link breadcrumbs__link--btn"
+      @click="$router.push('/')"
```

Comment:
I think I would expect as a user to go only one level back with this button as it says "back"